behaviour/graql/language/match.feature

```diff
@@ -994,6 +1196,119 @@ Feature: Graql Match Clause
       | SUS |
 
 
+  Scenario: when all instances of a type own a single attribute, matches with and without `!==` partition them
```

Comment:
hmm intent not obvious at first reading

Reply:
Yeah this test was migrated directly from IT but it seems needlessly bulky - we can break it down.

Reply:
Deleted redundant test (test steps fully contained in other tests)